.github/workflows/test.yml

```diff
@@ -26,15 +26,24 @@ jobs:
       fail-fast: false
       matrix:
         config:
+          # On Windows,
+          #
+          # * for R >= 4.2, both the MSVC toolchain and the GNU toolchain should
+          #   work. Since our main support is on MSVC, we mainly test MSVC here.
+          #   Also, at least one GNU case should be included so that we can
+          #   detect when something gets broken. 
+          # * for R < 4.2, the MSVC toolchain must be used to support
+          #   cross-compilation for the 32-bit. 
+          # 
```

Comment:
```suggestion
          # On Windows,
          #
          # * for R >= 4.2, both the MSVC toolchain and the GNU toolchain should
          #   work. Since we primarily support MSVC, we focus on testing MSVC here.
          #   Also, at least one GNU case should be included so that we can
          #   detect when something gets broken. 
          # * for R < 4.2, the MSVC toolchain must be used to support
          #   cross-compilation to 32-bit. 
          # 
```